tensorboard/webapp/app_routing/internal_utils.ts

```diff
@@ -74,6 +101,12 @@ export function getExperimentIdsFromRouteParams(
       }
       return [DEFAULT_EXPERIMENT_ID];
     }
+    case RouteKind.CARD: {
+      const typedParams = params as CompareRouteParams;
+      return parseCardExperimentStr(typedParams.experimentIds).map(
+        ({id}) => id
+      );
```

Comment:
If there's only one experiment id, we can just return it and no need to create empty name. The function above can be removed then.

suggestion: 
```
const experimentIds = (params as CompareRouteParams).experimentIds;
if (experimentIds.indexOf(',') < 0) {
   return [experimentIds]
}

return parseCompareExperimentStr(typedParams.experimentIds).map(
        ({id}) => id
      );
```

Reply:
Thank you for the suggestion! Done.